src/guide/effects.mdx

```diff
@@ -0,0 +1,498 @@
+  anchors.fill: img
+  source: img
+  maskSource: mask
+  maskEnabled: true  // This must be true for the mask to work
+  maskInverted: false  // You can use this to invert the masked area
+
+  // These provide antialiasing for the edges of the mask and make it look much better
+  maskSpreadAtMin: 1
+  maskThresholdMin: 0.5
+}
+```
+
+### Blurring
+
+Blurring is also done using the @@QtQuick.Effects.MultiEffect component. It is done similarly
+to shadows, however instead of setting the @@QtQuick.Effects.MultiEffect.shadowEnabled
+property to `true`, you set the @@QtQuick.Effects.MultiEffect.blurEnabled property to `true`.
+The blur effect can be customised with the @@QtQuick.Effects.MultiEffect.blur,
+@@QtQuick.Effects.MultiEffect.blurMax and @@QtQuick.Effects.MultiEffect.blurMultiplier properties.
+
+For example:
+
+```qml
+@@QtQuick.Image {
+  id: img
+
+  source: "/path/to/img"
+  visible: false
+}
+
+@@QtQuick.Effects.MultiEffect {
+  anchors.fill: img
+  source: img
+  blurEnabled: true
+  blur: 1  // This affects the radius of the blur (a real between 0 and 1)
+  blurMax: 32  // This affects the max radius of the blur (i.e. the radius when `blur` is 1)
+  blurMultiplier: 1  // A multiplier for the blur radius (0 is not multiplied)
+}
+```
+
+### Stacking effects
+
+The advantage of using a @@QtQuick.Effects.MultiEffect is that you can apply multiple effects
+to a component in a single pass. You can do this by simply setting that effect's corresponding enabled
+property to `true` and configuring it as shown previously.
+
+For example, the snippet below applies colourisation, shadow and blur:
+
+```qml
+@@QtQuick.Image {
+  id: img
+
+  source: "/path/to/img"
+  visible: false
+}
+
+@@QtQuick.Effects.MultiEffect {
+  anchors.fill: img
+  source: img
+
+  // Apply colourisation
+  colorization: 1
+  colorizationColor: "#abc4d3"
+
+  // Apply shadow
+  shadowEnabled: true
+  shadowBlur: 1
+  shadowColor: "#000000"
+
+  // Apply blur (`blurMax` also affects shadow)
+  blurEnabled: true
+  blur: 1
+  blurMax: 32
+}
+```
+
+### Using MultiEffect as a layer effect
+
+The @@QtQuick.Effects.MultiEffect component can also be used as a layer effect instead of a
+separate visual item. For example, you can replicate the previous using a layer effect:
+
+> [!NOTE]
+> The source component should be visible and have @@QtQuick.Item.layer.enabled set to `true`.
+> Additionally, the @@QtQuick.Effects.MultiEffect does not need to have a size or the
+> @@QtQuick.Effects.MultiEffect.source property set.
+
+```qml
+@@QtQuick.Image {
+  id: img
+
+  source: "/path/to/img"
+
+  layer.enabled: true
+  layer.effect: @@QtQuick.Effects.MultiEffect {
+    // Apply colourisation
+    colorization: 1
+    colorizationColor: "#abc4d3"
+
+    // Apply shadow
+    shadowEnabled: true
+    shadowBlur: 1
+    shadowColor: "#000000"
+
+    // Apply blur (`blurMax` also affects shadow)
+    blurEnabled: true
+    blur: 1
+    blurMax: 32
+  }
+}
+```
```

Comment:
Ok you did touch on this, but you should swap how its done throughout the guide page. Layer.effect is the preferred method, and I'd mention the alternative around here.